org.scala-ide.sdt.core/src/org/scalaide/util/eclipse/FileUtils.scala

```
@@ -104,6 +121,14 @@ object FileUtils {
     resources.find(prefix isPrefixOf _.getFullPath)
   }
 
+  private def containerResourceForPath(location: String, prefix: IPath = Path.EMPTY): Option[IContainer] = {
+    containerResourceForPath(Path.fromOSString(location), prefix)
+  }
+
+  private def containerResourceForPath(location: IPath, prefix: IPath): Option[IContainer] = {
+    val resources = Try(workspaceRoot.findContainersForLocationURI(URIUtil.toURI(location))).getOrElse(Array())
```

I don't like that an exception is ignored. Which exceptions do you expect here?